docs/source/migrating-from-gateway.mdx

```diff
@@ -1,24 +1,24 @@
 ---
 title: Gateway Migration Guide
-subtitle: Migrating to the Apollo Router from @apollo/gateway
-description: Learn how to migrate a federated graph from @apollo/gateway  to the Apollo Router without any changes to your subgraphs.
+subtitle: Migrating to the GraphOS Router from @apollo/gateway
```

Comment:
Available in both Router Core and GraphOS Router (for all occurrences on this page)